samples/client/petstore/python-asyncio/petstore_api/models/format_test.py

```diff
@@ -321,27 +321,27 @@ def binary(self, binary):
         self._binary = binary
 
     @property
-    def date(self):
-        """Gets the date of this FormatTest.  # noqa: E501
+    def _date(self):
+        """Gets the _date of this FormatTest.  # noqa: E501
 
 
-        :return: The date of this FormatTest.  # noqa: E501
+        :return: The _date of this FormatTest.  # noqa: E501
         :rtype: date
         """
-        return self._date
+        return self.__date
 
-    @date.setter
-    def date(self, date):
-        """Sets the date of this FormatTest.
+    @_date.setter
+    def _date(self, _date):
```

Comment:
If you are trying to avoid a conflict with `date`, you should put the underscore at the end not at the beginning since it usually means that an argument/method is private (https://docs.python-guide.org/writing/style/#we-are-all-responsible-users). It is also recommended by PEP8: https://www.python.org/dev/peps/pep-0008/#function-and-variable-names.

Reply:
@xavierhardy it's not my change. It appears after regenerate this sample client.

Reply:
https://github.com/swagger-api/swagger-codegen/pull/7956/files < this introduced the change but without regenerating samples